cvmfs/swissknife_check.cc

```diff
@@ -369,12 +389,14 @@ bool CommandCheck::Find(const catalog::Catalog *catalog,
     }
 
     // Check if the chunk is there
-    if (check_chunks_ &&
+    if (check_chunks_ && entry_needs_check_ &&
```

Comment:
We use the duplicates map also for the individual chunks of the entry (further down the method in `if (entries[i].IsChunkedFile())`).